developer-docs-site/docs/reference/faq.md

```diff
@@ -0,0 +1,328 @@
+---
+title: "Frequently Asked Questions"
+slug: "aptos-faq"
+---
+
+# Aptos Frequently Asked Questions
+
+This page answers common questions by subject. To add your own answers, create pull requests using the *Edit this page* link at the bottom. To ask a question for inclusion here, file an [Aptos documentation](https://github.com/aptos-labs/aptos-core/issues/new?assignees=clay-aptos&labels=bug%2Cdocumentation&template=documentation_bug_report.md&title=%5BDocs%5D) issue.
+
+## Aptos Labs
+
+### What is Aptos Labs?
+
+Aptos Labs is a startup that is building the safest and most production-ready blockchain. Our goal is to create universal and fair access to decentralized assets. The founding team at Aptos consists of an impressive group of researchers, engineers, designers and strategists.
+
+Read this blog post, “[The Genesis of Aptos](https://medium.com/aptoslabs/the-genesis-of-aptos-ff98d86e9445)” by our co-founder and CEO.
+
+### What does the word “Aptos” mean?
+
+Aptos is Ohlone for “The People” and encompasses our mission and ethos for why we build.
+
+### What is your website and blog?
+
+- Aptos Labs Website: [https://aptoslabs.com](https://aptoslabs.com/)
+- Aptos Labs Blog: [https://medium.com/aptoslabs](https://medium.com/aptoslabs)
+
+### Where can I find information about the Aptos Labs team?
+
+See [https://www.linkedin.com/company/aptoslabs](https://www.linkedin.com/company/aptoslabs)
+
+### Where is the Aptos Labs white paper?
+
+https://aptos.dev/aptos-white-paper/aptos-white-paper-index
+
+### Where is your developer documentation?
+
+See: https://aptos.dev/
+
+(This site. :-))
+
+### What makes Aptos unique compared to alternative blockchains?
+
+We have a specific focus on building a safe, scalable and upgradeable blockchain with credible neutrality.  [](https://aptos.dev/transactions/interacting-with-the-aptos-blockchain)
+
+We have a head start, as we are the original creators, researchers, designers, and builders of Diem, the blockchain that was first built to serve this purpose. We use the Move language, the safe and reliable language originally developed for Diem.
+
+Join the Discord channel [Aptos Move language channel](https://discord.com/channels/945856774056083548/955573698868432896) and get involved. 
+
+Also see [Interacting with Aptos Blockchain](https://aptos.dev/guides/interacting-with-the-aptos-blockchain) document. 
+
+What types of projects would be interesting and uniquely suited for Aptos blockchain platform?
+
+We are building a general purpose blockchain development platform, so the many use cases range from decentralized finance to gaming, NFTs, Decentralized autonomous organizations (DAOs), and beyond!
+
+### *What changed from Diem to Aptos?*
```

Comment:
this is an over simplification... I don't think we should entertain this question without a substantial effort to really enumerate in this space... but that's really hard since it was 30-40 people working intensely over 12 months that made this transition.